### DISCUSSION TITLE
Missing translation key for "Energy Distribution Today" in energy card title

### DISCUSSION THREAD

**Discussion by dmedina2015** [home-assistant/frontend#10668] [Localization] [answered]
When grid information is configured inside Energy Dashboard, a new card shows up in the Overview Panel. Its called "Energy distribution today" by default (see image):
![image](https://user-images.githubusercontent.com/16577563/142709144-94fc3681-9595-482d-aabf-533eaa59485b.png)

I'm translating all missing energy-related keys to Portuguese, but this card title has no translation key. Instead, it is hard-coded inside src/panels/lovelace/common/generate-lovelace-config.ts 
```
if (grid && grid.flow_from.length > 0) {
      areaCards.push({
      title: "Energy distribution today",
      type: "energy-distribution",
      link_dashboard: true,
});
```

Could someone please create a translation key for it? 

Thank you. Regards.

PS: I would code it by myself and post a PR if I knew how to do it, but i'm not a developer. I'm just a happy user trying to help improve HomeAssistant.

**Comment by spacegaier** [comment, marked as answer]
This PR will fix that: https://github.com/home-assistant/frontend/pull/10696

**Reply by dmedina2015** [reply]
Thank you @spacegaier for your help.
